src/components/structures/RoomDirectory.js

```diff
@@ -519,31 +519,54 @@ export default class RoomDirectory extends React.Component {
                                 MatrixClientPeg.get().getHomeserverUrl(),
                                 room.avatar_url, 32, 32, "crop",
                             );
-        return (
-            <tr key={ room.room_id }
+        return [
+            <div
```

Comment:
Let's tweak the code style here slightly, so that the closing bracket for elements spread across multiple lines is on a line by itself, and the class is on the same line as the opening bracket, like this:

```jsx
<div className="blah"
    key="..."
    onClick="..."
>
    ... children ...
</div>
```

I think that will better match the prevailing code style for such things.

Reply:
✓ done